src/main/java/frc/robot/drive.java

```diff
@@ -0,0 +1,65 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+
+import edu.wpi.first.wpilibj.XboxController;
+/**
+ * Add your docs here.
+ */
+public class drive {
+    public static CANSparkMax L1, L2, R1, R2;
+    public static XboxController driveController;
+    public static DoubleSolenoid rDriveShift, lDriveShift;
+    public static int PCM_CAN_ID = 19;
+    public static boolean driveMechToggle;
+    public static void initDrive(){
```

Comment:
Nit: Can we put a newline here clean separation between variable definition and the start of method definitions.